.travis.yml

```diff
@@ -27,10 +27,23 @@ matrix:
       os: osx
       osx_image: xcode10.1
       before_install:
-      - gem update --system
-      - gem install bundler
+        - gem update --system
+        - gem install bundler
       script:
-      - bundle exec pod lib lint Workflow.podspec --allow-warnings
+        - bundle exec pod lib lint Workflow.podspec --allow-warnings
+    - language: swift
```

Comment:
Could we name these jobs to make differentiating between the two Xcode builds easier?

Reply:
Done!